app/src/utils/convertState.ts

```diff
@@ -0,0 +1,25 @@
+export enum RequestStates {
+  DELETED,
```

Comment:
Lets follow up with Genita tomorrow to double check that this list of states is correct.